src/main/java/uk/gov/companieshouse/api/accounts/validation/IntangibleAssetsValidator.java

```diff
@@ -329,16 +384,38 @@ private void validatePresenceOfSingleYearFilerFields(IntangibleAssetsResource in
             subResourceInvalid = true;
         }
 
-        if(intangibleAssetsResource.getCost() != null && intangibleAssetsResource.getCost().getAtPeriodEnd() == null) {
+        if(intangibleAssetsResource.getCost() != null
+                && intangibleAssetsResource.getCost().getAtPeriodEnd() == null) {
             addError(errors, valueRequired, getJsonPath(intangibleSubResource, COST_AT_PERIOD_END));
             subResourceInvalid = true;
         }
 
+        if(intangibleAssetsResource.getAmortisation() != null
+                && intangibleAssetsResource.getAmortisation().getAtPeriodStart() != null) {
+            addError(errors, unexpectedData, getJsonPath(intangibleSubResource, COST_AT_PERIOD_START));
+            subResourceInvalid = true;
+        }
+
+        if(intangibleAssetsResource.getAmortisation() != null
+                && hasAmortisationFieldsSet(intangibleAssetsResource.getAmortisation())
+                && intangibleAssetsResource.getAmortisation().getAtPeriodEnd() == null) {
+            addError(errors, valueRequired, getJsonPath(intangibleSubResource, AMORTISATION_AT_PERIOD_END));
+            subResourceInvalid = true;
+        }
+
         if(subResourceInvalid) {
             invalidSubResource.add(intangibleSubResource);
         }
     }
 
+    private boolean hasAmortisationFieldsSet(Amortisation amortisation) {
+        return Stream.of(amortisation.getChargeForYear(),
```

Comment:
Do we need to include amortisation at period start in this stream?

Reply:
Discussed and agreed it stays out until Multi-Year validation is implemented.